macros/edr/tests/test_utils/create_elementary_test_table.sql

```diff
@@ -1,22 +1,18 @@
-{% macro create_elementary_test_table(database_name, schema_name, test_name, table_type, sql_query, is_temp_table=False) %}
+{% macro create_elementary_test_table(database_name, schema_name, test_name, table_type, sql_query) %}
     {% if execute %}
-        {% set temp_table_name = elementary.table_name_with_suffix(test_name, "__" ~ table_type) %}
+        {% set temp_table_name = elementary.table_name_with_suffix(test_name, "__" ~ table_type ~ elementary.get_timestamped_table_suffix()) %}
```

Comment:
Actually this also creates a problem that those tables will not override because tables will keep getting created and not deleted.

Reply:
Resolved [here](https://github.com/elementary-data/dbt-data-reliability/pull/502/commits/bb53906d995811d030ae57edb31d9aabb8acec08).